scripts/lib.sh

```diff
@@ -96,11 +96,6 @@ build_hub() {
 	popd >>/dev/null
 }
 
-get_kata_hash_from_tag() {
```

Comment:
Hey sorry, I just see that this is also used in https://github.com/kata-containers/packaging/blob/master/release/publish-kata-image.sh#L62  could you move `get_kata_hash` here and replace and use it in that file ?

Reply:
good catch!, fixed, thanks